### PR TITLE
Support single-platform evaluations for home manager

### PR DESCRIPTION
**Pull request by grahamc**

### CODE REVIEW

File: src/cli/mod.rs [hunk 1]
```diff
@@ -87,6 +87,20 @@ pub(crate) struct FlakeHubPushCli {
...
+    /// This flag is intended to limit the scope of evaluations which are too large to complete on one machine.
+    /// This flag should NOT be used to paper over evaluation errors across different architectures.
+    ///
+    /// Please do not turn this flag on without opening an issue to decide if it applies to your scenario.
+    #[clap(
+      long,
+      env = "FLAKEHUB_PUSH_SINGLE_SYSTEM_EVALUATION",
+      value_parser = EmptyBoolParser,
+      default_value_t = false
+  )]
```
Comment (cole-h) [inline src/cli/mod.rs:101]: nit:

```suggestion
    )]
```

File: src/flake_info.rs [hunk 2]
```diff
@@ -236,6 +241,10 @@ impl FlakeMetadata {
     }
 
     pub async fn outputs(&self, include_output_paths: bool) -> Result<FlakeOutputs> {
+        if self.single_system_evaluation {
+            return Ok(FlakeOutputs(serde_json::json!({})));
+        }
```
Comment (cole-h) [inline src/flake_info.rs:246]: This should probably have a different name than "single system evaluation" -- maybe `--large-evaluation` or something?

Comment (cole-h) [reply src/flake_info.rs:246]: Or probably better: `--skip-outputs` or similar

Comment (grahamc) [reply src/flake_info.rs:246]: originally itwas called `gigantic` ?

File: src/cli/mod.rs [hunk 3]
```diff
@@ -87,6 +87,20 @@ pub(crate) struct FlakeHubPushCli {
     )]
     pub(crate) error_on_conflict: bool,
 
+    /// Single-system evaluation.
+    ///
+    /// This flag is intended to limit the scope of evaluations which are too large to complete on one machine.
+    /// This flag should NOT be used to paper over evaluation errors across different architectures.
+    ///
+    /// Please do not turn this flag on without opening an issue to decide if it applies to your scenario.
```
Comment (cole-h) [inline src/cli/mod.rs:95]: comment should probably be reworded

Comment (grahamc) [reply src/cli/mod.rs:95]: bah.

Comment (grahamc) [reply src/cli/mod.rs]: fixed

**Review by cole-h — APPROVED** [review]

**Pull request merged**